apps/ledger-live-mobile/src/screens/Account/ListHeaderComponent.tsx

```diff
@@ -125,6 +128,19 @@ export function getListHeaderComponents({
 
   const stickyHeaderIndices = empty ? [] : [0];
 
+  const [oldestEditableOperation] = account.pendingOperations
+    .filter(pendingOperation => {
+      return isEditableOperation(account, pendingOperation);
+    })
+    .sort(
+      (a, b) => a.transactionSequenceNumber! - b.transactionSequenceNumber!,
```

Comment:
I think this is supposed to be handle by the `getStuckAccountAndOperation` helper now, isn't it ?